Resources/Prototypes/_Goobstation/Research/civilianservices.yml

```
@@ -0,0 +1,17 @@
+
+- type: technology
+  id: MedicalDefense
+  name: research-technology-medical-defense
+  icon:
+    sprite: _Goobstation/Objects/Weapons/Guns/Cannons/syringe_gun.rsi
+    state: rapid-syringe-gun
+  discipline: CivilianServices
+  tier: 2
+  cost: 10000
+  recipeUnlocks:
+  - RapidSyringeGun
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)